File: Hook/app/src/main/java/com/kuanquan/hook/WorkApplication.java
package com.kuanquan.hook;

import android.app.Application;

import com.kuanquan.hook.instrumentation.ActivityThreadHook;
import com.kuanquan.hook.reportSizeConfigurations.ReportSizeConfigurationsHook;

public class WorkApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        ActivityThreadHook.hook();
        ReportSizeConfigurationsHook.hook();
    }
}

File: VideoCover/app/src/main/java/com/kuanquan/videocover/WorkApplication.kt
package com.kuanquan.videocover

import android.app.Application
import android.content.Context
import androidx.camera.camera2.Camera2Config
import androidx.camera.core.CameraXConfig

class WorkApplication: Application(), CameraXConfig.Provider {

    override fun attachBaseContext(base: Context?) {
        super.attachBaseContext(base)
    }

    override fun onCreate() {
        super.onCreate()
    }

    override fun getCameraXConfig(): CameraXConfig {
        return Camera2Config.defaultConfig()
    }

}